### PR TITLE
Update perform-migration-from-svn-to-git.md

### PR DESCRIPTION
**Pull request by itsayushaarav**
added command for migrating specific set of revisions

### CODE REVIEW

**Comment by PRMerger18** [comment]
@itsayushaarav : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by Eresa12** [comment]
commencer

**Comment by steved0x** [comment]
#sign-off

**Comment by PRMerger18** [comment]
@steved0x: I'm sorry - only the author of this article, @vijayma, can sign off on your changes. But we do have an exception process - if you are on the Microsoft content or product team for this product area, you can ask the PR review team to review and merge it by sending mail to the techdocprs alias.

File: docs/repos/git/perform-migration-from-svn-to-git.md [hunk 1]
```diff
@@ -176,6 +176,16 @@ git for-each-ref --format='%(refname)' refs/remotes | % { $_.Replace('refs/remot
 >
 > This command can take a few minutes to several hours depending on the size of the SVN repository. Upon completion, you will have a Git checkout of your repository.
 
+### Migrate only specific revisions
+
+When not specified, git-svn clone will migrate all the revisions from first commit (r1) to HEAD. Should there any case arise, that one needs to migrate only specific set of revisions, the command for git-svn clone, should be appended with an option of -r.
```
Comment (Court72) [inline docs/repos/git/perform-migration-from-svn-to-git.md:181]:
```suggestion
When not specified, `git-svn clone` will migrate all of the revisions from the first commit (r1) to HEAD. If you need to migrate only a specific set of revisions, the command for `git-svn clone` should be appended with an option of `-r`.
```

**Comment by PRMerger8** [comment]
@Court72 : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Pull request merged**